src/main/java/au/com/sealink/quicktravel/client/retrofit/UserAgentInterceptor.java

```diff
@@ -0,0 +1,23 @@
+package au.com.sealink.quicktravel.client.retrofit;
+
+import java.io.IOException;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+
+public class UserAgentInterceptor implements Interceptor {
+    private volatile String userAgent;
+
+    public void setUserAgent(String userAgent) {
+        this.userAgent = userAgent;
+    }
+
+    @Override
+    public okhttp3.Response intercept(okhttp3.Interceptor.Chain chain) throws IOException {
+        Request request = chain.request();
+        request = request.newBuilder()
+                .addHeader("User-Agent", this.userAgent)
```

Comment:
Should we set some sort of sensible default in case the client does not set a useragent?

Reply:
Done